charts/fe-charts/values.yaml

```diff
@@ -74,4 +74,5 @@ configmap:
   data: {}
 
 secret:
+  isSealedSecrets: false
```

Comment:
Default value if no value is specified (backwards compatibility w/ b2xportal, mobile, etc.)